src/Connection/PostObjects.php

```diff
@@ -19,6 +19,34 @@ class PostObjects {
 	 */
 	public static function register_connections() {
 
+		register_graphql_connection(
+			[
+				'fromType'       => 'RootQuery',
+				'toType'         => 'ContentNode',
+				'queryClass'     => 'WP_Query',
+				'resolveNode'    => function( $id, $args, $context, $info ) {
+					return DataSource::resolve_post_object( $id, $context );
+				},
+				'fromFieldName'  => 'contentNodes',
+				'connectionArgs' => self::get_connection_args(
+					[
+						'contentTypes' => [
+							'type'        => [ 'list_of' => 'PostTypeEnum' ],
+							'description' => __( 'The Types of content to filter', 'wp-graphql' ),
+						],
+					],
+					null
+				),
+				'resolve'        => function( $source, $args, $context, $info ) {
```

Comment:
Suggestion: Could this be moved together with the other connection wrappers in the `DataSource` class?

Reply:
Ah yes. I originally pulled it in here because I thought I was going to need to customize and use `$resolver->setQueryArg()`, but that ended up not being the case. I reverted to use the existing `DataSource::resolve_post_objects_connection();`

Good call! 😄